spring-cloud-gcp-samples/spring-cloud-gcp-secretmanager-sample/src/main/resources/templates/index.html

```diff
@@ -40,12 +40,12 @@
 <h1>Secret Manager Demo with Spring Cloud GCP</h1>
 
 <div class="panel">
-    <h3>Secret Manager Property Source</h3>
-    At the bootstrap phase, we loaded the following secret into the application context:
-    <br/>
-    <br/>
-    <b>application-secret:</b> <i>[[${applicationSecret}]]</i><br/>
-    <b>my-application-secret:</b> <i>[[${myApplicationSecret}]]</i>
+  <h3>Secret Manager Property Source</h3>
+  At the bootstrap phase, we loaded the following secret into the application context:
+  <br/>
+  <br/>
+  <b>application-secret:</b> <i>[[${applicationSecret}]]</i><br/>
+  <b>my-application-secret:</b> <i>[[${myApplicationSecret}]]</i>
```

Comment:
Could you rename those to "Application secret from @Value" and "Application secret from @ConfigurationProperties"?

Reply:
done.